static/app/utils/profiling/renderers/gridRenderer.tsx

```diff
@@ -0,0 +1,138 @@
+import {mat3, vec2} from 'gl-matrix';
+
+import {FlamegraphTheme} from '../flamegraph/FlamegraphTheme';
+import {getContext, measureText, Rect} from '../gl/utils';
+
+export function getIntervalTimeAtX(configToPhysicalSpace: mat3, x: number): number {
+  const logicalToPhysical = mat3.fromScaling(
+    mat3.create(),
+    vec2.fromValues(window.devicePixelRatio ?? 1, window.devicePixelRatio ?? 1)
+  );
+  const physicalToConfigSpace = mat3.invert(mat3.create(), configToPhysicalSpace);
+
+  const logicalToConfigSpace = mat3.multiply(
+    mat3.create(),
+    physicalToConfigSpace,
+    logicalToPhysical
+  );
```

Comment:
Perhaps I'm misunderstanding this part but should this be
```suggestion
  const logicalToConfigSpace = mat3.multiply(
    mat3.create(),
    logicalToPhysical,   
    physicalToConfigSpace
  );
```

The way I understand it is as follows

```
A = configToPhysical
B = A^-1 = physicalToConfig
C = logicalToPhysical
D = C x B = logicalToPhysical x physicalToConfig = logicalToConfig
```

But what you have here is
```
D = B x C = physicalToConfig x logicalToPhysical = logicalToConfig
```

Reply:
Oh yeah, definitely correct order here. It doesnt make a difference because logicalToPhysical is an identity matrix (never has any rotations or translations), so it works either way, but nice catch :)!